src/GafferUI/StandardConnectionGadget.cpp

```diff
@@ -186,6 +186,35 @@ void StandardConnectionGadget::doRenderLayer( Layer layer, const Style *style )
 		return;
 	}
 
+	// If we're missing the srcNodule, the source node is either hidden or we're dealing with an AuxiliaryConnection
+	if( !srcNodule() )
+	{
+		const Plug *srcPlug = dstNodule()->plug()->getInput();
+		const GraphGadget *graphGadget = parent<GafferUI::GraphGadget>();
+
+		if( const NodeGadget *srcNodeGadget = graphGadget->nodeGadget( srcPlug->node() ) )
+		{
+			if( srcNodeGadget->getVisible() )
+			{
+				const Box3f b = srcNodeGadget->transformedBound();
+				const Box2f srcNodeFrame( V2f( b.min.x, b.min.y ), V2f( b.max.x, b.max.y ) );
+
+				const M44f m = dstNodule()->fullTransform( srcNodeGadget );
```

Comment:
Do we really need to duplicate the logic from `setPositionsFromNodules()` here? What's wrong with `m_dstPos` and `m_dstTangent`?